src/sagemaker/feature_store/feature_processor/_config_uploader.py

```diff
@@ -12,7 +12,7 @@
 # language governing permissions and limitations under the License.
 """Contains classes for preparing and uploading configs for a scheduled feature processor."""
 from __future__ import absolute_import
-from typing import Callable, Dict, Tuple, List
+from typing import Callable, Dict, Optional, Tuple, List
```

Comment:
Instead of using `Optional[...]`, it would be better to use `... | None`. That requires having this at the top in order to support Python < 3.10:
```py
from __future__ import annotations
```

Reply:
The `Optional` is used a lot in the code base, I'll address it in another PR